src/api/models/Profile.ts

```diff
@@ -24,7 +24,7 @@ export class Profile extends Bookshelf.Model<Profile> {
         }
     }
 
-    public static async fetchByName(value: string, withRelated: boolean = true): Promise<Profile> {
+    public static async fetchByName(value: string = '', withRelated: boolean = true): Promise<Profile> {
         if (withRelated) {
```

Comment:
why this? if value is null/undefined we should have thrown an error during validation allready